cmd/server/assets/realm.html

```diff
@@ -21,6 +21,21 @@ <h1>Realm settings</h1>
       Find or edit the settings for <strong>{{$realm.Name}}</strong> below.
     </p>
 
+    {{if not $realm.EnableENExpress}}
+      <div class="card mb-3">
+        <div class="card-header">
+          ENX - Exposure Notifications Express
```

Comment:
```suggestion
          EN Express - Exposure Notifications Express
```

We say "EN Express" everywhere else except these titles.